### PR TITLE
Adds support for uWSGI/nginx deployment.

### PR DESCRIPTION
**Pull request by oceanzus**

### CODE REVIEW

**Comment by DanielJMaher** [comment]
@oceanzus Who is the person to check this... @FBRTMaka ??

**Comment by oceanzus** [comment]
@DanielJMaher Sure, Matt can be the reviewer.

File: nginx.conf [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        location @WSGI {
+            include uwsgi_params;
+            uwsgi_pass unix:/tmp/ooi-ui/uwsgi.sock;
+        }
+    }
+    
+    server {
+
+        # Running port
+        listen 4000;
```
Comment (FBRTMaka) [inline nginx.conf]: @oceanzus , this good on 4000 for the UI?

**Comment by oceanzus** [comment]
Yes.

**Comment by FBRTMaka** [comment]
Word

**Pull request merged**